src/main/java/duke/task/Parser.java

```diff
@@ -0,0 +1,107 @@
+package duke.task;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+import duke.task.Deadline;
+import duke.task.DukeException;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.task.Todo;
+
+public class Parser {
+    public static Date getDateFromUser(String input) {
+        return parseDate(input, "dd/MM/yyyy HHmm");
+    }
+
+    public static Date getDateFromFile(String input) {
+        return parseDate(input, "EEE MMM dd HH:mm:ss Z yyyy");
+    }
+
+    public static Date parseDate(String input, String pattern) {
+        Date date = new Date();
+
+        try {
+            date = new SimpleDateFormat(pattern).parse(input);
+        } catch (Exception e) {
+            System.err.println(e.getMessage());
+            System.exit(1);
+        }
```

Comment:
For Duke, when an Exception occur, the application should not terminate but continue running instead. I think an exception should be handled instead of exiting the application.